OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -623,7 +623,131 @@ hasStateStorage() const
 //-----------------------------------------------------------------------------
 // INTEGRATION
 //-----------------------------------------------------------------------------
-///____________________________________________________________________________
+/**
+* Integrate the equations of motion for the specified model, given the current
+* state (at which the integration will start) and a finalTime. Make sure to
+* use state.setTime(initialTime) to specify a starting time before calling
```

Comment:
Consider giving an example in this comment block, and also getting the link to `SimTK::State::setTime()` to work.